spaceship/lib/spaceship/tunes/iap_list.rb

```diff
@@ -58,6 +58,13 @@ def status
       def edit
         attrs = client.load_iap(app_id: application.apple_id, purchase_id: self.purchase_id)
         attrs[:application] = application
+
+        if attrs["addOnType"] == Spaceship::Tunes::IAPType::RECURRING
+          raw_pricing_data = client.load_recurring_iap_pricing(app_id: application.apple_id,
+                                                               purchase_id: self.purchase_id)
+          attrs["pricingData"] = raw_pricing_data
```

Comment:
This is not very fancy, I suppose changing the constructor of IAPDetails to take two distinct hashes is not doable?

Reply:
I didn't like it either, but in `Base::initialize` it says: ``Do not override `initialize` in your own models.``